src/features/pages/Home/Stats.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  RingProgress,
+  Text,
+  SimpleGrid,
+  Paper,
+  Center,
+  Group,
+  rem,
+  Container,
+} from "@mantine/core";
+import { IconArrowUpRight, IconArrowDownRight } from "@tabler/icons-react";
+
+const icons = {
+  up: IconArrowUpRight,
+  down: IconArrowDownRight,
+};
+
+const data = [
+  {
+    label: "Page views",
+    stats: "456,578",
+    progress: 80,
+    color: "teal",
+    icon: "up",
+  },
+  {
+    label: "New users",
```

Comment:
pokaż jak wygląda wyśrodkowany tekst z punktu 2. Wydaje mi się, że można to ładnie wyśrodkować

Reply:
![1](https://github.com/Best-offer-finder/frontend/assets/80382822/a0f83476-4d71-4cea-b178-cb8415a47c31)

Coś takiego

Reply:
bo tylko tekst w największym kontenerze jest wyśrodkowany.
Jak dodasz wyśrodkowanie w tytułach i wartościach liczbowych, to będzie ładnie wyglądać

Reply:
Ustaw stałą szerokość tych tekstów na np. width: 150px. Dodaj dodatkowo text-align:center i powinno być ładnie

Reply:
![image](https://github.com/Best-offer-finder/frontend/assets/80382822/15325a4f-1a24-41a8-a458-faf304541ec0)
No mi nadal mniej się to podoba ale whatever moge wrzucić tak

Reply:
bo za szeroki tekst dałeś xd pewnie 100px lepiej by wyglądało